boards/DIM/Test/Src/Test_StateMachine.cpp

```diff
@@ -244,7 +244,46 @@ TEST_F(
 
 TEST_F(
     DimStateMachineTest,
-    check_drive_mode_is_broadcasted_over_can_in_drive_state)
+    check_deadzones_for_mapped_paddle_position_in_drive_state_DIM_8)
+{
+    // <= 5% maps to 0 %
+    get_raw_paddle_position_fake.return_val = 4;
+    App_SharedStateMachine_Tick(state_machine);
+    ASSERT_EQ(
+        0,
+        App_CanTx_GetPeriodicSignal_MAPPED_PADDLE_POSITION(can_tx_interface));
+    get_raw_paddle_position_fake.return_val = 5;
+    App_SharedStateMachine_Tick(state_machine);
+    ASSERT_EQ(
+        0,
+        App_CanTx_GetPeriodicSignal_MAPPED_PADDLE_POSITION(can_tx_interface));
+    get_raw_paddle_position_fake.return_val = 6;
+    App_SharedStateMachine_Tick(state_machine);
+    ASSERT_NE(
+        0,
+        App_CanTx_GetPeriodicSignal_MAPPED_PADDLE_POSITION(can_tx_interface));
+
+    // >= 95% maps to 100%
+    get_raw_paddle_position_fake.return_val = 94;
+    App_SharedStateMachine_Tick(state_machine);
+    ASSERT_NE(
+        100,
+        App_CanTx_GetPeriodicSignal_MAPPED_PADDLE_POSITION(can_tx_interface));
+    get_raw_paddle_position_fake.return_val = 95;
+    App_SharedStateMachine_Tick(state_machine);
+    ASSERT_EQ(
+        100,
+        App_CanTx_GetPeriodicSignal_MAPPED_PADDLE_POSITION(can_tx_interface));
+    get_raw_paddle_position_fake.return_val = 96;
+    App_SharedStateMachine_Tick(state_machine);
+    ASSERT_EQ(
+        100,
+        App_CanTx_GetPeriodicSignal_MAPPED_PADDLE_POSITION(can_tx_interface));
+}
```

Comment:
This doesn't seem like part of this PR?

Reply:
Ah yes forgot to leave a comment but as i was adding the specification ID I realized we forgot to write a test case for this particular ID. Slightly out-of-scope but I just quickly wrote a test for it.